api/user.go

```diff
@@ -394,6 +394,41 @@ func Login(c *Context, w http.ResponseWriter, r *http.Request, user *model.User,
 
 	http.SetCookie(w, sessionCookie)
```

Comment:
We still use the original cookie.  I've added a second to manage the different teams.  MMSID is your current team and MMSIDMU is the list of possible teams.